index.html

```diff
@@ -54,58 +55,351 @@ <h1> Space Ripples Reveal Big Bang’s Smoking Gun</h1>
+        cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the multiverse.</p>
+      <p>So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.</p>
+      <p>We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.</p>
+      <p>“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”</p>
+      <div id='end-body' class="grid">
+        <div class='end-body-1'>
+          A version of this article appears in print on March 18, 2014, Section A, Page 1 of the New York edition with the headline: Space Ripples Reveal Big Bang’s Smoking Gun. <a href="https://www.parsintl.com/publication/the-new-york-times/">Order
+            Reprints</a> | <a href="https://www.nytimes.com/section/todayspaper">Today’s Paper </a>| <a href="https://www.nytimes.com/subscription?campaignId=48JQY">Subscribe</a>
+        </div>
+        <div class="end-body-2">
+          <button type="button" name="button">Read 615 Comments</button>
         </div>
-      </div>
-      <div class='cmmtbutton'>
-        <button type="button" name="button">Read Comments</button>
-      </div>
-      <div class='solinks'>
         <div id=slinks>
           <i class="fab fa-facebook-f"></i>
           <i class="fab fa-twitter"></i>
           <i class="fas fa-envelope"></i>
           <i class="fas fa-share"></i>
+          <i class="fas fa-bookmark"></i>
+          <i class='fas fa-comment'></i>
         </div>
       </div>
     </div>
   </main>
-  <footer>
-  </footer>
-  <aside class="editorPicks">
-    Editors’ Picks
 
+  <div>
+    <div id="container" class='grid'>
+      <div class="grid">
+        <hr>
+        <h3> Suggested newsletters for you</h3>
+        <div id="nls-container" class='grid'>
+          <div class="nl-1-span">
+            <span>AS NEEDED</span>
+          </div>
+          <div class="nl-2-span">
+            <span>WEEKDAYS</span>
+          </div>
+          <div class="nl-3-span">
+            <span>DAILY</span>
+          </div>
+          <div class="nl-1-img">
+            <img src="pictures/nl1.png" alt="">
+          </div>
+          <div class="nl-2-img">
+            <img src="pictures/nl2.png" alt="">
+          </div>
+          <div class="nl-3-img">
+            <img src="pictures/nl3.png" alt="">
+          </div>
+          <div class="nl-1-b">
+            <b>BREAKING NEWS</b>
+          </div>
+          <div class="nl-2-b">
+            <b>Coronavirus Briefing</b>
+          </div>
+          <div class="nl-3-b">
+            <b>The Morning</b>
+          </div>
```

Comment:
## Reduce the spacing between elements
- [ ] There are far much margin and spacing in-between the elements of this section than necessary. kindly reduce the spacing and ensure the contents in these sections are centred on the page as is expected.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/amiraabouhadid/project1/page-styling/index.html)|
|:--:|:--:|
|![suggested-section](https://user-images.githubusercontent.com/22345961/108413987-b843fe00-723c-11eb-91dd-37517e209ccd.png)|![suggested-section-clone](https://user-images.githubusercontent.com/22345961/108414004-be39df00-723c-11eb-8ca9-baef13abe9d4.png)|